src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -26,30 +26,66 @@ ALTER SEQUENCE pg_catalog.pg_dist_placement_placementid_seq RESTART 100000;
 SELECT nextval('pg_catalog.pg_dist_groupid_seq') AS last_group_id \gset
 SELECT nextval('pg_catalog.pg_dist_node_nodeid_seq') AS last_node_id \gset
 -- Create the necessary test utility function
-CREATE FUNCTION master_metadata_snapshot()
+CREATE FUNCTION activate_node_snapshot()
     RETURNS text[]
     LANGUAGE C STRICT
     AS 'citus';
-COMMENT ON FUNCTION master_metadata_snapshot()
-    IS 'commands to create the metadata snapshot';
+COMMENT ON FUNCTION activate_node_snapshot()
+    IS 'commands to activate node snapshot';
 -- Show that none of the existing tables are qualified to be MX tables
 SELECT * FROM pg_dist_partition WHERE partmethod='h' AND repmodel='s';
  logicalrelid | partmethod | partkey | colocationid | repmodel | autoconverted
 ---------------------------------------------------------------------
 (0 rows)
 
--- Show that, with no MX tables, metadata snapshot contains only the delete commands,
--- pg_dist_node entries and reference tables
-SELECT unnest(master_metadata_snapshot()) order by 1;
-                                                                                                                                                                                                                       unnest
+-- Since password_encryption default has been changed to sha from md5 with PG14
+-- we are updating it manually just for consistent test results between PG versions.
+ALTER SYSTEM SET password_encryption TO md5;
+SELECT pg_reload_conf();
+ pg_reload_conf
+---------------------------------------------------------------------
+ t
+(1 row)
+
+SELECT pg_sleep(0.1);
+ pg_sleep
+---------------------------------------------------------------------
+
+(1 row)
+
+ALTER ROLE CURRENT_USER WITH PASSWORD 'dummypassword';
+-- Show that, with no MX tables, activate node snapshot contains only the delete commands,
+-- pg_dist_node entries, pg_dist_object entries and roles.
+SELECT unnest(activate_node_snapshot()) order by 1;
+                                                                                                                                                                             unnest
 ---------------------------------------------------------------------
+ CREATE SCHEMA IF NOT EXISTS public AUTHORIZATION postgres
```

Comment:
if these tests produce different outputs in the enterprise, we may consider adding `WHERE .. not ilike '%GRANT%'`